examples/web-chat/src/MessageInput.tsx

```diff
@@ -63,7 +68,7 @@ export default function MessageInput(props: Props) {
     >
       <Row align="center">
         <Fill>
-          <TextInput value={inputText} />
```

Comment:
Because `ui` lib weirdly works it does not rely on any external state. If `active` props is true then after `onMessage` is called it automatically cleans input's value. 
In case `active` is not true it won't clean it - that's what was happening.